src/Compilers/CSharp/Test/Emit/Attributes/AttributeTests_NullableMembers.cs

```diff
@@ -0,0 +1,10 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests
+{
+    public class AttributeTests_NullableMembers : CSharpTestBase
```

Comment:
>AttributeTests_NullableMembers [](start = 17, length = 30)

Do we need this file?

Reply:
Added tests.

---
In reply to: [293590761](https://github.com/dotnet/roslyn/pull/36398#discussion_r293590761) [](ancestors = 293590761)